yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher.java

```diff
@@ -56,12 +56,20 @@
     /**
      * Configured from UI
      */
-    protected JNLPLauncher jnlpLauncher = new JNLPLauncher();
+    protected transient JNLPLauncher jnlpLauncher;
```

Comment:
Add `@Deprecated` annotation and `@deprecated because properties moved to fields` into javadoc.

JFYI transient will load value from XStream deserialisation and then forget about serialisation. That makes OldDataMonitor in manage jenkins happy.